client/my-sites/plans/current-plan/feature-not-included-card/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { Card, Gridicon } from '@automattic/components';
+import { Component } from 'react';
+
+import './style.scss';
+
+class FeatureNotIncludedCard extends Component {
+	render() {
+		const { illustration, title, text } = this.props;
+
+		return (
+			<Card className="feature-not-included-card__card">
+				<img
+					className="feature-not-included-card__illustration"
+					alt={ title }
+					src={ illustration }
+				/>
+				<div>
+					<Gridicon className="feature-not-included-card__cross" icon="cross" size={ 24 } />
```

Comment:
Maybe something to tackle in a follow-up, but I am not sure if a cross is helpful when we want users to upgrade. Maybe a plus sign (`+`) might be better so we're more clearly communicating "You can also get these" rather than "You don't have this".

<img width="373" alt="Screenshot 2023-02-06 at 13 10 48" src="https://user-images.githubusercontent.com/3376401/216957451-b44638ff-98b0-4273-bd04-ac4277c8ca50.png">